user-org-service/app/controllers/usermanagement/UserController.java

```diff
@@ -21,7 +21,7 @@
    */
   public CompletionStage<Result> createUser() {
     startTrace("createUser");
-    CompletionStage<Result> response = createHandelRequest(request());
+    CompletionStage<Result> response = handleRequest(request(),null,"createUser");
```

Comment:
Can we think of the operation be set, rather than pass? BaseController can expose a protected operationName field, which the derived classes can set. Also, if there is httpExecutionContext going to be null, please create an overload of handleRequest - rather than passing null.

Reply:
in place of null, we will pass validation function (lambda expression) right now since validation is not there in this pr so I place null over here.